### ISSUE TITLE
[BUG] 重复频繁弹出Popup导致闪退.weak global reference table overflow

### ISSUE THREAD

**Issue by BaY1251** [CommunityToolkit/Maui#1448]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

不断循环打开Popup导致闪退
10-10 13:05:23.686	rockchip rk3288	Error	7853	DEBUG	Abort message: 'art/runtime/indirect_reference_table.cc:132] JNI ERROR (app bug): weak global reference table overflow (max=51200)'
    r0 00000000  r1 00000740  r2 00000006  r3 00000008
    r4 852f7978  r5 00000006  r6 852f7920  r7 0000010c
    r8 00000000  r9 0000000a  sl 0000224d  fp 85a71e00
    ip 00000000  sp 852f6af8  lr a83fb857  pc a83fe0c0  cpsr 600d0010
10-10 13:05:23.681	rockchip rk3288	Error	7853	DEBUG	signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
10-10 13:05:23.681	rockchip rk3288	Error	7853	DEBUG	pid: 1684, tid: 1856, name: Thread Pool Wor  >>> com.ainuo.magic <<<
10-10 13:05:23.681	rockchip rk3288	Error	7853	DEBUG	ABI: 'arm'
10-10 13:05:23.681	rockchip rk3288	Error	7853	DEBUG	Revision: '0'
10-10 13:05:23.681	rockchip rk3288	Error	7853	DEBUG	Build fingerprint: 'Android/rk3288/rk3288:7.1.2/NHG47K/root09111143:userdebug/test-keys'
10-10 13:05:23.681	rockchip rk3288	Error	7853	DEBUG	*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
10-10 13:05:23.608	rockchip rk3288	Warning	198		debuggerd: handling request: pid=1684 uid=10059 gid=10059 tid=1856
10-10 13:05:23.607	rockchip rk3288	Error	1684	libc	Fatal signal 6 (SIGABRT), code -6 in tid 1856 (Thread Pool Wor)
10-10 13:05:23.607	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427] 
10-10 13:05:23.607	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427] 
10-10 13:05:23.607	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]         1 of android.widget.LinearLayout
10-10 13:05:23.607	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]         1 of android.graphics.drawable.shapes.RectShape
10-10 13:05:23.607	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]         1 of android.content.pm.ApplicationInfo
1 

....

10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     17165 of android.view.RenderNode (17165 unique instances)
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]   Summary:
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51190: 0x36f31f00 android.graphics.Path
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51191: 0x36ecc7c0 android.graphics.Path
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51192: 0x36e89b60 crc643f46942d9dd1fff9.FormattedStringExtensions_FontSpan
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51193: 0x36e3c610 crc643f46942d9dd1fff9.FormattedStringExtensions_FontSpan
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51194: 0x36e2e1e0 crc643f46942d9dd1fff9.FormattedStringExtensions_FontSpan
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51195: 0x36e12808 android.graphics.RectF
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51196: 0x36d35390 mono.android.view.View_OnLayoutChangeListenerImplementor
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51197: 0x36bece38 android.view.ViewGroup$LayoutParams
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51198: 0x36d0b6d0 android.view.ViewGroup$LayoutParams
10-10 13:05:23.604	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]     51199: 0x36ecc7a0 android.graphics.Path
10-10 13:05:23.603	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427]   Last 10 entries (of 51200):
10-10 13:05:23.603	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427] weak global reference table dump:
10-10 13:05:23.603	rockchip rk3288	Error	1684	art	art/runtime/runtime.cc:427] JNI ERROR (app bug): weak global reference table overflow (max=51200)

### Expected Behavior

支持频繁弹出

### Steps To Reproduce

使用定时器 频繁自动弹出和退出 Popup. 在不确定的时间(1小时或者1天)后,App闪退

### Link to public reproduction project repository

暂无

### Environment

```markdown
- .NET MAUI CommunityToolkit:6.0.0
- OS:Android 7.1
- .NET MAUI:7.0.92
```

**Comment by pictos** [comment]
@BaY1251, please translate your report to english, that way we can understand and help you (:

**Comment by cat0363** [comment]
For your reference, I have placed the verification code I created below.
https://github.com/cat0363/MauiComm-IssuePopupLeak.git

My verification code caused an Out of memory exception, but since the verification terminal was in a sleep state, I am verifying it again. The code that I reproduced may be different from the code that caused this issue, so it would be better to have code that can reproduce this issue.
